.appveyor.yml

```diff
@@ -9,8 +9,8 @@ cache:
 
 environment:
     global:
-        CONDA_CHANNELS: conda-forge
-        CONDA_DEPENDENCIES: pip setuptools wheel cython mock six biopython networkx joblib matplotlib scipy vs2015_runtime pytest mmtf-python GridDataFormats hypothesis pytest-cov codecov chemfiles tqdm
+        CONDA_CHANNELS: conda-forge rdkit
```

Comment:
can use the conda-forge rdkit packaging right?

Reply:
I completely forgot it's now available through conda-forge!